google/resource_dns_record_set_test.go

```diff
@@ -140,6 +140,42 @@ func testAccCheckDnsRecordSetExists(resourceType, resourceName string) resource.
 	}
 }
 
+func testAccCheckDnsRecordSetValid(resourceType, resourceName string) resource.TestCheckFunc {
```

Comment:
this function doesn't seem to get called from anywhere, and the inputs don't seem to get used. I think you meant to make this its own unit test- take a look at https://github.com/terraform-providers/terraform-provider-google/blob/master/google/resource_dataproc_cluster_test.go#L24 for an example of how to do this.

Reply:
forgot to mark this as done